src/index.css

```diff
@@ -35,29 +58,160 @@ ul {
 
 .books-list {
   display: flex;
-  width: 250px;
-  justify-content: space-between;
+  height: 7rem;
+  width: 70%;
+  background-color: #fff;
+  padding: 5px 1.688rem;
+  margin-left: 4.25rem;
```

Comment:
- [ ] Please as mentioned by the previous reviewer kindly center your book list. This will improve your style in your app will look better 😁 😉

<img width="941" alt="Screen Shot 2022-11-12 at 02 01 35" src="https://user-images.githubusercontent.com/74501523/201446542-66a5f24e-4135-408e-87b2-819a57177302.png">